tests/test_qdrant_client.py

```
@@ -1757,6 +1758,35 @@ def test_timeout_propagation():
+    # for. The sync ApiClient.request already promotes that value.
+    from qdrant_client.http.api_client import AsyncApiClient
+
+    captured: dict = {}
+
+    def _capture(method, url, **kwargs):
+        captured["kwargs"] = kwargs
+        return type("DummyRequest", (), {"headers": {}})()
+
+    async_client = AsyncApiClient(host="http://localhost:6333")
+    with patch.object(async_client._async_client, "build_request", _capture):
+        with pytest.raises(Exception):
```

Ruff (0.16.1)

[warning] Do not assert blind exception: `Exception`

(B017)